src/android/SecureStorage.java

```diff
@@ -75,10 +76,32 @@ public boolean execute(String action, CordovaArgs args, final CallbackContext ca
         }
         if ("init".equals(action)) {
             String service = args.getString(0);
+            JSONObject options = args.getJSONObject(1);
+            String packageName = options.getString("packageName");
+
+            Context ctx = null;
+            
+            // Solves #151. By default, we use our own ApplicationContext
+            // If packageName is provided, we try to get the Context of another Application with that packageName
+            if (packageName.equals("null")) {
```

Comment:
Please move null check in getPackageContext.

Reply:
Done.